### PR TITLE
Modify find command to allow find substrings in words

### PR DESCRIPTION
**Pull request by Skaty**
Author: @AHaliq

### CODE REVIEW

File: src/main/java/seedu/address/commons/util/StringUtil.java [hunk 1]
```diff
@@ -10,11 +10,19 @@
  */
 public class StringUtil {
     public static boolean containsIgnoreCase(String source, String query) {
-        String[] split = source.toLowerCase().split("\\s+");
-        List<String> strings = Arrays.asList(split);
+        List<String> strings = listifyString(source);
```
Comment (advaypal) [inline src/main/java/seedu/address/commons/util/StringUtil.java:13]: These changes are in the other PR too. Please don't duplicate code like this across PRs

File: src/main/java/seedu/address/model/ModelManager.java [hunk 2]
```diff
@@ -150,4 +150,25 @@ public String toString() {
         }
     }
 
+    private class NameSubstringQualifier implements Qualifier {
```
Comment (advaypal) [inline src/main/java/seedu/address/model/ModelManager.java:153]: Same thing here

**Review by advaypal — CHANGES_REQUESTED** [review]
All the changes of this PR(except the tests I think) are in the other PR too. Please don't do something like this again. For now you can add the test in that PR, along with the other comments I made there, and can then close this PR.

**Comment by Skaty** [comment]
Refer to #30

**Pull request closed without merging**